backend/tracim_backend/views/core_api/user_controller.py

```diff
@@ -610,6 +614,36 @@ def get_user_messages(
             )
         )
 
+    @hapic.with_api_doc(tags=[SWAGGER_TAG__USER_EVENT_ENDPOINTS])
+    @check_right(has_personal_access)
+    @hapic.input_path(UserIdPathSchema())
+    @hapic.input_query(MessageSummaryQuerySchema())
+    @hapic.output_body(MessageSummarySchema())
+    def get_user_messages_summary(self, context, request: TracimRequest, hapic_data: HapicData):
```

Comment:
Don't forget to annotate the return type.

Reply:
done.